src/main/java/org/broadinstitute/hellbender/tools/copynumber/utils/annotatedregion/AnnotatedRegionParser.java

```diff
@@ -0,0 +1,88 @@
+package org.broadinstitute.hellbender.tools.copynumber.utils.annotatedregion;
+
+import com.google.common.collect.Sets;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.utils.tsv.DataLine;
+import org.broadinstitute.hellbender.utils.tsv.TableColumnCollection;
+import org.broadinstitute.hellbender.utils.tsv.TableReader;
+import org.broadinstitute.hellbender.utils.tsv.TableWriter;
+import org.testng.collections.Lists;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.*;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+
+public class AnnotatedRegionParser {
```

Comment:
Even if we are going to eventually write a Feature-based tool, I think we should make this a LocatableCollection when that PR is in.  I don't like having custom static methods for reading and writing each type of "Feature".  Could you file an issue?